html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+
+  <h1>Coding Form!</h1>
+
+  <label for="email">Email<br></label>
+  <input type="text" id="email" required minlength="15" maxlength="30" size="10">
+  <div>Why is coding cool</div>
+  <label for="why"></label>
```

Comment:
the label should have the text in it so that you can click on the text and put focus on the input